app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -129,39 +116,58 @@
             app:layout_constraintEnd_toStartOf="@id/sign_in_button"
             app:layout_constraintHorizontal_bias="0.0"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="@id/sign_in_button" />
+            app:layout_constraintTop_toBottomOf="@id/terms_and_cond_checkbox"
+            app:layout_constraintVertical_bias="0.99" />
 
         <android.widget.Button
             android:id="@+id/sign_in_button"
             style="@style/SignInButtonStyle"
             android:layout_width="@dimen/sign_in_button_width"
             android:layout_height="@dimen/sign_in_button_height"
             android:layout_marginLeft="@dimen/signin_margin_left"
+            android:layout_marginTop="@dimen/login_buttons_margin_top"
             android:drawableEnd="@drawable/ic_arrow_forward_white_24dp"
             android:drawableTint="@color/white"
+            android:focusable="true"
             android:gravity="center"
             android:paddingEnd="@dimen/sign_in_button_paddingEnd"
             android:text="@string/sign_in"
             android:textSize="@dimen/sign_in_text_size"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintHorizontal_bias="1.0"
-            app:layout_constraintStart_toStartOf="parent" />
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/terms_and_cond_checkbox"
+            app:layout_constraintVertical_bias="0.99" />
     </androidx.constraintlayout.widget.ConstraintLayout>
 
     <Button
         android:id="@+id/forgot_password_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginLeft="@dimen/forgot_pass_margin_left"
+        android:layout_marginLeft="@dimen/forgot_pass_horizontal_margin"
         android:layout_marginTop="@dimen/forgot_pass_margin_top"
-        android:layout_marginRight="@dimen/forgot_pass_margin_right"
+        android:layout_marginRight="@dimen/forgot_pass_horizontal_margin"
+        android:layout_marginBottom="@dimen/forgot_pass_margin_bottom"
         android:backgroundTint="@android:color/transparent"
         android:stateListAnimator="@null"
         android:text="@string/forgot_password"
         android:textSize="@dimen/sign_in_text_size"
-        app:layout_constraintEnd_toEndOf="@id/info_container"
-        app:layout_constraintStart_toStartOf="@id/info_container"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/info_container" />
+
+    <ProgressBar
```

Comment:
Move this in the middle of the InfoContainer or in the middle of the screen. Right now it is very small and not very visible.